### PR TITLE
Functional statistics and charts

### LINKED ISSUE #2989: Warning on ticket statistics
When going to the ticket statistics page, it is possible to see the following warning:

image

### PR DESCRIPTION
**Pull request by bgptr**
This diff refactors staking statistics and charts components to functional component using hooks and CSS modules.

Part of #2439 and #2438
Closes #2989

Also, it improves their layout on small screens. Some before and after screenshots:

shot
shots2
shots3
shots4
shots5

### CODE REVIEW

File: app/components/views/TicketsPage/StatisticsTab/Statistics.module.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
+.isRow {
+  display: flex;
+  flex-direction: row;
+}
+.myTicketsCharts {
+  -webkit-user-select: text;
+}
+.myTicketsCharts {
```
Comment (amassarwi) [inline app/components/views/TicketsPage/StatisticsTab/Statistics.module.css:8]: myTickets prefix can be ditched.

**Review by amassarwi — APPROVED** [review]
LGTM overall, added two minor inline comments.

File: app/components/views/TicketsPage/StatisticsTab/Statistics.module.css (hunk 1 above)
Comment (bgptr) [reply app/components/views/TicketsPage/StatisticsTab/Statistics.module.css:8]: Sure, that's a good point. Thx

**Review by alexlyp — APPROVED** [review]

**Comment by alexlyp** [comment]
whoops the indicators module PR conflicted.  sorry

**Comment by bgptr** [comment]
np 😀 resolved

**Pull request merged**